tools/coverage.sh

```diff
@@ -1,21 +0,0 @@
-#!/bin/bash
-# This runs cargo test and creates test coverage data, as well as a test result report, currently this requires the 'nightly' rust toolchain.
-# Run this in the project root, and cargo2junit and grcov binaries (do `cargo install cargo2junit grcov`)
-# Result files will be placed in ./reports
-
-PROJECT_NAME_UNDERSCORE="up_streamer_rust"
-RUSTFLAGS="--cfg uuid_unstable -Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off -Zpanic_abort_tests -Cpanic=abort"
-RUSTDOCFLAGS="-Cpanic=abort"
-TEMP=`mktemp --directory`
-
-
-mkdir -p $TEMP/reports
-
-cargo test $CARGO_OPTIONS -- -Z unstable-options --format json | cargo2junit > $TEMP/results.xml
```

Comment:
-Z and following args have dependency on nightly builds, which makes the rest of the script unusable/not work